apps/cloud-log/package.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "@yoda/cloud-log",
+  "version": "1.0.0",
+  "description": "global cloud log on-off switch",
+  "main": "app.js",
+  "metadata": {
+    "skills": [
```

Comment:
Should present `extapp: false` :(

Reply:
this field doesn't exist nor work now. If an app is an ext app determined by config `/etc/yoda/app-loader-config.json`.

Reply:
so, currently what should I do to make sure it is ext app. XD

Reply:
By default all apps are ext-app, only light app shall be configured as light-app.